packages/js/components/src/section/section.tsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import { createElement } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { Level } from './context';
+
+type SectionProps = {
+	// The wrapper component for this section. Optional, defaults to `div`. If passed false, no wrapper is used. Additional props
+	// passed to Section are passed on to the component.
```

Comment:
```suggestion
	/** The wrapper component for this section. Optional, defaults to `div`. If passed false, no wrapper is used. Additional props passed to Section are passed on to the component. */
```

Reply:
It seems that we need to use JSDoc comment style (`/**...*/`) so vscode can show the hover information. 

![Screen Shot 2023-01-06 at 11 11 32](https://user-images.githubusercontent.com/4344253/210922442-7589f04c-d443-41b4-aba5-695d2fbe6373.png)

Reply:
TIL -- I had no idea! 

Thank you for the change!